packages/perseus/src/widgets/interactive-graphs/graphs/components/angle-indicators.test.ts

```diff
@@ -104,3 +108,66 @@ describe("shouldDrawArcOutside", () => {
         expect(getClockwiseAngle(coords)).toBe(45);
     });
 });
+
+describe("getWholeAngleMeasure", () => {
+    test("should return 0 for no angle", () => {
+        const coords: [Coord, Coord, Coord] = [
+            [0, 0],
+            [0, 0],
+            [0, 0],
+        ];
+        const vertex = coords[1];
+
+        expect(getWholeAngleMeasure(coords, vertex)).toBe(0);
+    });
+
+    test("should return 270 for a reflex right angle", () => {
+        const coords: [Coord, Coord, Coord] = [
+            [0, 0],
+            [0, 1],
+            [1, 0],
+        ];
+        const vertex = coords[1];
+
+        expect(getWholeAngleMeasure(coords, vertex)).toBe(270);
+    });
+
+    test("should not return decimals for angle", () => {
+        const coords: [Coord, Coord, Coord] = [
+            [0, 0],
+            [7, 0.5],
+            [12.5, 2.5],
+        ];
+        const vertex = coords[1];
+        expect(getWholeAngleMeasure(coords, vertex)).toBe(184);
+        expect(getWholeAngleMeasure(coords, vertex)).not.toBe(
+            184.08561677997488,
+        );
```

Comment:
This second assertion will always pass if the first assertion passes, so I think we don't need it. If you want to document what the value _would_ be if it weren't rounded, I think a comment would be fine.

```suggestion
```